ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
+$icon-width: 24px;
+$icon-margin-right: 2rem;
+$question-padding: 1.5rem;
+
+.faq-block__icon {
+    font-size: 30px;
+    margin-right: $icon-margin-right;
+}
+
+.faq-block__question {
+    margin: 0;
+}
+
+.faq-block__text {
+    white-space: pre-wrap;
+}
+
+.mat-expansion-panel {
+    border-top: 2px solid $border-color;
+    font-family: Untitled-Sans-Regular;
+    font-size: 24px;
```

Comment:
should we use rem for font-size as it's done for other studies?

Reply:
Agreed. Added at 8e4e176298eb4955d2838fd1abe0c06da8731fdb